src/Helpers/MiscHelper.js

```diff
@@ -62,7 +62,7 @@ export function formatDate(date = new Date()) {
     const toFormat = new Date(date);
     if (toFormat instanceof Date && !isNaN(toFormat)) {
         const year = toFormat.getFullYear();
-        const month = toFormat.getMonth();
+        const month = toFormat.getMonth() + 1; // month is zero indexed
```

Comment:
OMG, I have never noticed that :+1: :face_with_head_bandage:

Reply:
To be fair it is really inintuitive, even getDate() to get the day from Date is 1 indexed